src/Nethermind/Nethermind.Runner/Ethereum/Steps/StartRpc.cs

```diff
@@ -30,8 +32,34 @@ public StartRpc(INethermindApi api)
         public async Task Execute(CancellationToken cancellationToken)
         {
             IJsonRpcConfig jsonRpcConfig = _api.Config<IJsonRpcConfig>();
+            IKeyStoreConfig keyStoreConfig = _api.Config<IKeyStoreConfig>();
             ILogger logger = _api.LogManager.GetClassLogger();
 
+
+            // Update the JWT secret path based on the data directory.
+            try
+            {
+                string defaultPath = "keystore/jwt-secret";
+                string newPath = Path.Combine(keyStoreConfig.KeyStoreDirectory, "jwt-secret");
+                if (string.IsNullOrEmpty(jsonRpcConfig.JwtSecretFile))
+                {
+                    logger.Warn("JsonRpcConfig JwtSecretFile is null or empty because of omission or human error.");
```

Comment:
The warning is redundant. JWT secret is not always needed. For example, for Aura networks.
```suggestion
```